src/acceptance-test/java/org/zalando/nakadi/repository/db/SchemaRepositoryTest.java

```diff
@@ -0,0 +1,70 @@
+package org.zalando.nakadi.repository.db;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.joda.time.DateTime;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeSchema;
+import org.zalando.nakadi.domain.EventTypeSchemaBase;
+import org.zalando.nakadi.domain.Version;
+import org.zalando.nakadi.utils.EventTypeTestBuilder;
+
+import java.util.List;
+
+public class SchemaRepositoryTest extends AbstractDbRepositoryTest {
+
+    private SchemaRepository repository;
+
+    public SchemaRepositoryTest() {
+        super(new String[]{"zn_data.event_type_schema"});
+    }
+
+    @Before
+    public void setUp() {
+        super.setUp();
+        repository = new SchemaRepository(template, mapper);
+    }
+
+    @Test
+    public void whenListVersionsListedOrdered() throws Exception {
+        final EventTypeSchemaBase schemaBase = new EventTypeSchemaBase(EventTypeSchemaBase.Type.JSON_SCHEMA, "schema");
+        final EventType eventType = EventTypeTestBuilder.builder()
+                .name("test_et_name_schemarepositorytest")
+                .schema(new EventTypeSchema(schemaBase, "1.0.2", DateTime.now()))
+                .build();
+        insertEventType(eventType);
+        insertSchema(eventType);
+        eventType.setSchema(new EventTypeSchema(schemaBase, "2.10.3", DateTime.now()));
+        insertSchema(eventType);
+        eventType.setSchema(new EventTypeSchema(schemaBase, "10.0.0", DateTime.now()));
```

Comment:
I'm afraid this test could fail if it runs too fast. If the precision of `DateTime.now()` is too low, it could sometimes generate the same value on lines 35, 39 or 41. Makes sense?